dxr/app.py

```diff
@@ -360,7 +361,6 @@ def _linked_pathname(path, tree_name):
 
     return components
 
-
```

Comment:
Let's keep the double line skip here. PEP 8 says 2 blank lines between top-level constructs.